src/unstoppableAPI.ts

```diff
@@ -48,6 +48,15 @@ export default class Udapi extends NamingService {
     return !!this.findMethod(domain);
   }
 
+  supportsRecords(domain?: string): boolean {
```

Comment:
Lets make this parameter required for all naming services for consistency. I don't see too much sense in making it optional for some. Unused arguments are common for abstract methods and inheritence.